VisualPinball.Unity/VisualPinball.Unity.Editor/Toolbox/ToolboxEditor.cs

```diff
@@ -205,21 +201,20 @@ private static bool CreateButton(string label, Texture icon, float iconSize, GUI
 			);
 		}
 
-		private static void CreateItem<TItem>(Func<Table, TItem> create, string actionName) where TItem : IItem
+		private static void CreateItem<TItem>(TableAuthoring tableAuth, Func<Table, TItem> create, string actionName) where TItem : IItem

```

Comment:
It's only `static` because previously wasn't using any members. Now I would suggest making it non-static and using `_table` directly instead of passing it through every call.